### ISSUE TITLE
Keyboard access to DataTables

### ISSUE THREAD

**Issue by vanch3d** [SAFeSEA/openEssayist-slim#2]
On ‘Your Draft : Key Words & Key Phrases’ the table columns cannot be sorted using the keyboard: the focus lands on the buttons, but pressing ‘enter’ does not activate the buttons. (Note the buttons can be operated with the screenreader.) 

On Your Draft : Key Words & Key Phrases, after operating the table sorting buttons the focus returns to the top of the page. If possible enable focus to remain on the button, or at least within the table. High priority.
